src/Marten.AspNetCore/Marten.AspNetCore.csproj

```diff
@@ -1,9 +1,9 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
-        <TargetFrameworks>net6.0;net7.0</TargetFrameworks>
+        <TargetFrameworks>net6.0;net7.0;net8.0</TargetFrameworks>
         <Description>Helpers for Marten-backed AspNetCore applications</Description>
-        <VersionPrefix>6.4.0</VersionPrefix>
+        <VersionPrefix>7.0.0-alpha.1</VersionPrefix>
```

Comment:
Have you considered using a `Directory.Build.props` where such common repeated properties can be defined once and are then re-used by MsBuild automatically?

Sou would just need to touch ~5 files instead of the 45 right now.

Reply:
@gfoidl All the primary Marten libraries all use `Directory.Build.props`. Only the plugins related csproj have specific versions set. It is definitely 5 and not 45 :-)

Reply:
I was just wondering to see the same / similar change to many csproj-files. 
So IMO `Directory.Build.props` could have more common properties, and together with [Central Package Management](https://learn.microsoft.com/en-us/nuget/consume-packages/central-package-management) such repeated changes could be minimized.

So nothing urgent or that like, just to save some work for @oskardudycz 😉.

> can have their own set of changes and related releases

Ah, OK. TIL they aren't versioned all together.

Reply:
Also note that when we do a major version update, we set a starting version for these plugin/additional projects in alignment with the major version. After this point, as I said, it can go through it own life cycle of changes/releases.

Reply:
I don't question that (was so in the first comment, but you told me the versioning strategy then).

So my point is*:
* set the target-frameworks in the Directory.Build.props, so an update goes only there and doesn't need to be done in all projects (each project still has to specify the property, but can use the variable defined in the Directory.Build.props)
* use central package management, so updates to packages go to one (central) place and are not spread to all csproj-files

\* this comment-thread is now in the wrong place, as the `VersionSuffix` is a you mention independ for each project

Reply:
@gfoidl, thank you for suggestions. I'll check the recent improvements around Central Package Management. The biggest challenge is that we try to minimise the project dependencies, and sometimes, we also need to have different package versions for different .NET versions. I'll check how much we can benefit from it. Good call. 👍

Reply:
From my experience CPM kind of sucked in pre .NET 8 and had many troubles. Starting with .NET 8 (actually already starting with the previews) it's really good to use and doesn't have any problems.

For me the only point to keep in mind is that `PrivateAssets`, etc. still needs to be set in the individual csproj that use such a package.

Reply:
Yes, that instability was stopping me from trying that further, but thanks for letting me know that improved. I'll give it a try in this PR or (probably) a dedicated one :+1:

Reply:
@gfoidl I added a dedicated issue for that: https://github.com/JasperFx/marten/issues/2804. I'll try to tackle that in the follow up PR.